alert.css

```diff
@@ -0,0 +1,95 @@
+:root{
+    --primary-color: #6366f1;
+    --alert-color:#ef4444;
+    --warning-color:#fbbf24;
+}
+.alert-container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    gap :0.5rem;
+}
+.alert-element{
+    width: 30%;
+   
+}
+
+/* On screens that are 992px or less */
+@media screen and (max-width: 992px) {
+    .alert-element{
+        width: 60%;
+    }
+  }
+  
+  /* On screens that are 600px or less, set the background color to olive */
+  @media screen and (max-width: 600px) {
+    .alert-element{
+        width: 90%;
+    }
+  }
+/* heading of alerts with white background color*/
+.alert-element:nth-child(1){
+    font-weight: bold;
+}
+/* alerts with white background color*/
+.alert-element:nth-child(2){
```

Comment:
The way you have designed / architectured your alerts will not work.

Consider the below scenario. User wants to use success alert (this is in accordance to what you have written)
```HTML
<div class="alert-element">success message</div>
```

but this will be a normal alert as you have given `alert-element:nth-child(4){}` to be success. 

Long story short, unless the user provides 4 children with class as `alert-element`, it will not show success alert.

Instead of using alert-element and nth-child combo, use separate classes for each of the types of alerts &mdash; `.alert-primary` : for primary color, `.alert-error` and so on

Reply:
I have made the advised changes.